autostart_windows.c

```diff
@@ -1,47 +1,53 @@
-#include <windows.h>
+#include <stdio.h>
+#include <shlobj.h>
 #include <stdlib.h>
 #include <stdint.h>
-#include <stdio.h>
 #include <string.h>
 #include <objbase.h>
-#include <shlobj.h>
+#include <windows.h>
 
-uint64_t CreateShortcut(char *shortcutA, char *path, char *args) {
-	IShellLink*   pISL;
-	IPersistFile* pIPF;
-	HRESULT       hr;
+uint64_t CreateShortcut(char *s, char *path, char *args)
```

Comment:
Seems like the only changes here are code style?

If you want to make code style changes, can you split these in a separate file? (Note, note sure I like all of these changes.)

Reply:
Alright